app/main.py

```diff
@@ -2,0 +1,21 @@
+def copy_file(command: str) -> None:
+    commands = command.split()
+    if (
+        len(commands) < 3
+        or commands[0] != "cp"
+        or commands[1] == commands[2]
+    ):
+        return
+    try:
+        with (
+            open(commands[1], "r") as source,
+            open(commands[2], "a") as target
+        ):
+            for line in source:
+                target.write(line)
+    except UnicodeError:
```

Comment:
You are only catching `UnicodeError` exceptions. However, other exceptions may occur during file operations, such as `FileNotFoundError` if the file doesn't exist, or `PermissionError` if you don't have the necessary permissions to read or write the file. It's a good practice to handle these exceptions as well.